unreleased.rst

```diff
@@ -8,3 +8,12 @@ This page contains release notes for the SDK.
 
 HEAD — ongoing
 --------------
+- [Sandbox] Added `--log-level` command line flag.
```

Comment:
Please rebase to fix the unreleased notes.

Reply:
done